commands/command_migrate.go

```diff
@@ -287,6 +294,8 @@ func init() {
 	importCmd := NewCommand("import", migrateImportCommand)
 	importCmd.Flags().BoolVar(&migrateVerbose, "verbose", false, "Verbose logging")
 	importCmd.Flags().StringVar(&objectMapFilePath, "object-map", "", "Object map file")
+	importCmd.Flags().BoolVar(&migrateNoRewrite, "no-rewrite", false, "Add new history without rewriting previous")
```

Comment:
I think that these are great short descriptions of what the `--no-rewrite` and `--message` flags do, which is exactly what is appropriate for these.

Could you also add a longer explanation of what these flags are in [`git-lfs-migrate(1)`](https://github.com/git-lfs/git-lfs/blob/master/docs/man/git-lfs-migrate.1.ronn)? The `docs/man` tree is used to automatically generate `commands/mancontent.go`, and this in turn is what is shown when you run `git lfs migrate --help`, or `git lfs help migrate`, for instance.

The existing prose should be a good place to start in terms of what level of detail is appropriate.